scripts/tests/run.mjs

```diff
@@ -92,8 +100,13 @@ if (browserstack) {
   // Create BrowserStack launchers
   const browsers = [];
   launchers.forEach(launcher => {
-    browsers.push(browserstackLauncher({ capabilities: { ...sharedCapabilities, ...launcher } }));
-  })
+    // Create browserName to show as a label in the progress bar reporter
+    let browserName = `${ launcher.browser ?? launcher.browserName ?? launcher.device ?? 'unknown' }${
```

Comment:
this string template is too complex. Let's simplify it.

Reply:
Nullish coalescing might not be helpful here as empty string would not be replaced with `'unknown'`. Is this intentional?

Reply:
It should be fine bacause I use the same code in the `@web/test-runner-browserstack` to get label, I think it has a bug in that package, so we need this workaround to make the label show up until they fix it.